test/functional/feature_segwit.py

```diff
@@ -138,11 +138,6 @@ def run_test(self):
         # unsigned with redeem script
         self.fail_accept(self.nodes[0], "mandatory-script-verify-flag", p2sh_ids[NODE_0][WIT_V0][0], False, witness_script(False, self.pubkey[0]))
         self.fail_accept(self.nodes[0], "mandatory-script-verify-flag", p2sh_ids[NODE_0][WIT_V1][0], False, witness_script(True, self.pubkey[0]))
-        # signed
```

Comment:
I think this whole block can be removed (`self.log.info("Verify default node can't accept any witness format txs before fork")` downwards). The previuos `self.fail_accept()` calls are testing transactions that are invalid both before and after the fork due to missing signatures.

Reply:
For the same reason as above, I think that including these tests is misleading for anyone reading the tests, since it implies that behaviour is different before and after segwit activation.

Reply:
Thanks! Reworded the comment in the last commit

Reply:
I still think these subtests should be moved to after segwit activation